sources/packages/web/src/components/layouts/Institution/sidebar/HomeSideBar.vue

```diff
@@ -1,49 +1,35 @@
 <template>
-  <v-navigation-drawer app class="body-background">
-    <v-list dense nav>
+  <v-navigation-drawer app class="body-background" permanent>
+    <v-list density="compact" nav>
       <v-list-item
         v-for="item in items"
         :key="item.label"
         @click="item.command"
-      >
-        <v-list-item-icon>
-          <v-icon>{{ item.icon }}</v-icon>
-        </v-list-item-icon>
-        <v-list-item-content>
-          <v-list-item-title>{{ item.label }}</v-list-item-title>
-        </v-list-item-content>
-      </v-list-item>
-      <v-list-item>
-        <v-list-item-content>
-          <v-list-item-title class="spaced-text text-muted"
-            >LOCATIONS</v-list-item-title
-          >
-        </v-list-item-content>
-      </v-list-item>
-      <v-list-item
-        v-for="location in locationsMenu"
-        :key="location.label"
-        @click="location.command"
-      >
-        <v-list-item-content>
-          <v-list-item-title
-            ><v-icon>{{ location.icon }}</v-icon>
-            {{ location.label }}</v-list-item-title
-          >
-          <v-list-item
-            v-for="locationItem in location?.items"
-            :key="locationItem"
-            @click="locationItem.command"
-          >
-            <v-list-item-icon>
-              <v-icon>{{ locationItem.icon }}</v-icon>
-            </v-list-item-icon>
-            <v-list-item-content>
-              <v-list-item-title>{{ locationItem.label }}</v-list-item-title>
-            </v-list-item-content>
-          </v-list-item>
-        </v-list-item-content>
-      </v-list-item>
+        :prepend-icon="item.icon"
+        :title="item.label"
+      />
+      <v-list density="compact" nav>
+        <v-list-subheader>Locations</v-list-subheader>
+        <v-list-item
+          v-for="location in locationsMenu"
+          :key="location.label"
+          @click="location.command"
+        >
+          <v-list-item-content>
+            <v-list-item-title
+              ><v-icon>{{ location.icon }}</v-icon>
+              {{ location.label }}</v-list-item-title
+            >
+            <v-list-item
+              v-for="locationItem in location?.items"
+              :key="locationItem"
+              :prepend-icon="locationItem.icon"
```

Comment:
👍

Reply:
just curious, is the ` <v-list-group>` working in the new beta version. if you didn't check, never mind, ignore the comment.

just wanted to check if we can achieve the below functioning with the new beta version,
![image](https://user-images.githubusercontent.com/77353155/176770310-8aac6994-19c1-4bc0-8ca9-ea26f89da1ba.png)

![image](https://user-images.githubusercontent.com/77353155/176770236-6a631137-c022-4ded-80ac-037e35a86f97.png)

Reply:
Let me take a look at this.

Reply:
no problem @andrewsignori-aot , we can merge the PR and maybe we can create a technical debt

Reply:
Changes as suggested and added the icons for expanding/collapsing.
![image](https://user-images.githubusercontent.com/61259237/176778156-a60ef9ed-f2ba-4607-8271-b9969e059ffd.png)